atomic_reactor/utils/imageutil.py

```diff
@@ -5,6 +5,91 @@
 This software may be modified and distributed under the terms
 of the BSD license. See the LICENSE file for details.
 """
+
+import functools
+from typing import Optional, Union
+
+from osbs.utils import ImageName
+
+from atomic_reactor import config
+from atomic_reactor import util
+
+
+def inspectable(image: Union[str, ImageName]) -> bool:
```

Comment:
Since this function is not a part of a class, the name itself - inspectable does not tell me much without that docs, so maybe something like  is_image_inspectable would improve readability

Reply:
+1, renamed